javav2/usecases/video_analyzer_application/Readme.md

```diff
@@ -49,18 +56,20 @@ An Amazon S3 bucket named **video[somevalue]**. Be sure to use this bucket name
 
 You must create an IAM role and a valid SNS topic. You need to reference these values in the **VideoDetectFaces** class. If you do not set these values, the application that you create does not work. For information, see [Configuring Amazon Rekognition Video](https://docs.aws.amazon.com/rekognition/latest/dg/api-video-roles.html).  
 
+**Note**: You must specify the AWS resource values in the **VideoApplicationResources** class. 
+
 
 ## Understand the AWS Video Analyzer application
 
-The AWS Video Analyzer application supports uploading a video (MP4 file) to an Amazon S3 bucket. After the video is uploaded, you can confirm that the video is located in the Amazon S3 bucket by choosing the **Show Video** button.
+The AWS Video Analyzer application supports viewing video (MP4 file) information that is located in an Amazon S3 bucket. That is, you can confirm that the video is located in the Amazon S3 bucket by choosing the **Show Video** button.
 
-![AWS Video Analyzer](images/pic1.png)
+![AWS Video Analyzer](images/video.png)
 
 To generate a report, enter an email address and choose **Analyze Video**. A mask is displayed to let the user know the report is being created. 
 
-![AWS Video Analyzer](images/pic2.png)
+![AWS Video Analyzer](images/videoReport.png)
 
-**Note** Depending upon the size of the video, this may take a few minutes. To test this functionality, keep the video under 20 seconds. Also, there can only be one video in the Amazon S3 bucket. 
+**Note:** Depending upon the size of the video, this may take a few minutes. To test this functionality, keep the video under 20 seconds. Also, there can only be one video in the Amazon S3 bucket. 
```

Comment:
```suggestion
**Note:** Depending on the size of the video, this might take a few minutes. To test this functionality, make the video less than 20 seconds long. Also, there can only be one video in the S3 bucket. 
```